tests/nvme/064

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0+
+# Copyright (C) 2025 Yi Zhang <yi.zhang@redhat.com>
+#
+# Test NVMe disk namespace management
+#
+
+. tests/nvme/rc
+
+DESCRIPTION="test NVMe disk namespace management"
+QUICK=1
+CAN_BE_ZONED=1
+
+requires() {
+	_nvme_requires
+	_have_fio
+}
+
+device_requires() {
+	_require_nvme_support_ns_mgt
+}
+
+test_device() {
+	echo "Running ${TEST_NAME}"
+
+	local ns_dev ctrl_dev nn tnvmcap lbaf lbads cntlid
+
+	ns_dev=${TEST_DEV##*/}
+        ctrl_dev=${ns_dev%n*}
+	nn=$(nvme id-ctrl /dev/"$ctrl_dev" | grep ^nn | awk '{ print $3}')
+	tnvmcap=$(nvme id-ctrl /dev/"$ctrl_dev" | grep tnvmcap | awk '{ print $3}')
+	lbaf=$(nvme id-ns "$TEST_DEV" | grep 'in use' | awk '{ print $2}')
+	lbads=$(nvme id-ns -H "$TEST_DEV" | grep 'in use' | awk '{ print $12}')
```

Comment:
Please use `--verbose` instead of the old `--human-readable`.

Reply:
Yeah, to check the Data Size, the --verbose cannot output that value of 512[1], it only output "ds":9[2], not sure if it's something we can improve on nvme-cli.
[1]
`LBA Format  1 : Metadata Size: 8   bytes - Data Size: 512 bytes - Relative Performance: 0x3 Degraded (in use)`
[2]
```
 "lbafs":[
    {
      "ms":0,
      "ds":9,
      "rp":1
    },
    {
      "ms":8,
      "ds":9,
      "rp":3
    },
    {
      "ms":0,
      "ds":12,
      "rp":0
    },
    {
      "ms":8,
      "ds":12,
      "rp":2
    },
    {
      "ms":64,
      "ds":12,
      "rp":3
    }
  ],

```

Reply:
Definitely something we can improve in nvme-cli. I understand that the verbose json output is lacking some information which is shown with stdout and verbose? Anyway, would you mind to open a bug report in nvme-cli so we don't forget about it. Thanks!

Reply:
yes, I've filed below issue to track it.
https://github.com/linux-nvme/nvme-cli/issues/2811